packages/css/src/plugins/increase-specificity.ts

```diff
@@ -0,0 +1,39 @@
+import type { Plugin } from 'postcss';
+
+const INCREASE_SPECIFICITY_SELECTOR = ':not(#\\9)';
```

Comment:
I understand this is inspired by [stylex's PR earlier today](https://github.com/facebook/stylex/pull/271), but is there any docs about this kind of CSS syntax? I still don't really understand how this is a valid selector 😆 

Also, why `\9` specifically (which I assume is U+0009?) and not a different escape character?

Reply:
Inspired by this old postcss plugin for the selector https://github.com/MadLittleMods/postcss-increase-specificity/tree/master and then stylex was just another signal that isn't not the worst idea ;-).

\9 is a tab character, pretty much just a grab bag of whatever works https://github.com/MadLittleMods/postcss-increase-specificity/tree/master